apps/meteor/ee/client/omnichannel/reports/utils/round.ts

```diff
@@ -0,0 +1,4 @@
+export const round = (value: number, decimals = 2) => {
+	const multiplier = Math.pow(10, decimals);
+	return Math.round(value * multiplier) / multiplier;
```

Comment:
Is it faster than `parseFloat(number.toFixed(2))`? I think so, but...

Reply:
![image](https://github.com/RocketChat/Rocket.Chat/assets/11577696/5c30889d-ecac-4ce7-8b19-fd391c07c659)